alf/algorithms/planning_algorithm.py

```diff
@@ -248,13 +255,13 @@ def _calc_cost_for_action_sequence(self, time_step: ActionTimeStep, state,
             time_step = time_step._replace(prev_action=action)
             time_step, state = self._dynamics_func(time_step, state)
             next_obs = time_step.observation
-            # Note: currently using (next_obs, action), might need to
-            # consider (obs, action) in order to be more compatible
-            # with the conventional definition of reward function
```

Comment:
why removing this comment? Still (next_obs, action). Perhaps comment the reward module that it's using next_obs and action?

Reply:
Added back and also added a note in the reward module.